<?xml version="1.0" encoding="UTF-8"?><magicdraw><comment classType="Comment" humanType="Comment" icon="index_files/icon_comment_841774062.jpg" id="_10_5_622020d_1132928308600_272588_5382" refElementId="_10_5_622020d_1132928308600_272588_5382"><annotatedElement humanName="Annotated Element" mode="s,e"><class classType="Class" humanType="Class" icon="index_files/icon_class_555864569.jpg" name="AsynchronousCloseException" refElementId="_9_5_f720368_1110473546643_22057_7871" refid="_9_5_f720368_1110473546643_22057_7871"/></annotatedElement><body humanName="Body" mode="s,e">Checked exception received by a thread when another thread closes the
channel or the part of the channel upon which it is blocked in an I/O
operation.

@version 1.9, 01/11/19
@since 1.4
</body><documentation humanName="Documentation"/><owner classType="Class" humanName="Owner" humanType="Class" icon="index_files/icon_class_555864569.jpg" mode="s,e" name="AsynchronousCloseException" refElementId="_9_5_f720368_1110473546643_22057_7871" refid="_9_5_f720368_1110473546643_22057_7871"/><owningElement classType="Class" humanName="Owning Element" humanType="Class" icon="index_files/icon_class_555864569.jpg" name="AsynchronousCloseException" refElementId="_9_5_f720368_1110473546643_22057_7871" refid="_9_5_f720368_1110473546643_22057_7871"/><TO_DO humanName="To Do" mode="s,e"/></comment></magicdraw>